core/src/main/scala/com/salesforce/op/stages/impl/preparators/SanityChecker.scala

```diff
@@ -465,10 +465,10 @@ class SanityChecker(uid: String = UID[SanityChecker])
 
       // Only perform Cramer's V calculation on columns that have an indicatorGroup and indicatorValue defined (right
```

Comment:
Update comment to match new name

Reply:
@leahmcguire I think this is the only one left. try grepping the codebase `indicatorGroup`